server/src/test/java/org/opensearch/indices/replication/OngoingSegmentReplicationsTests.java

```diff
@@ -74,7 +74,8 @@ public void setUp() throws Exception {
         ShardId testShardId = primary.shardId();
 
         // This mirrors the creation of the ReplicationCheckpoint inside CopyState
-        testCheckpoint = new ReplicationCheckpoint(testShardId, primary.getOperationPrimaryTerm(), 0L, 0L);
+        testCheckpoint = new ReplicationCheckpoint(testShardId, primary.getOperationPrimaryTerm(), 0L, 0L, "Lucene95");
+        testCheckpoint2 = new ReplicationCheckpoint(testShardId, primary.getOperationPrimaryTerm(), 0L, 0L, "Lucene94");
```

Comment:
nit: Can we generate these constants from actual codecs (or helper classes). Is there any utility which provides an older version of codec